json_minify/test_json_minify.py

```diff
@@ -83,5 +125,28 @@ def test_3(self):
     def test_4(self):
         self.template(4)
 
+    def _space_template(self, in_string, expected):
+        actual = json_minify(in_string, strip_space=False)
+        self.assertEqual(actual, expected)
+
+    def test_space_keeping(self):
+        self._space_template('', '')
+        self._space_template('\n', '\n')
+        self._space_template(' \r\n ', ' \r\n ')
+        self._space_template('\t', '\t')
```

Comment:
All the white space tests were basically just for me to fix my initial bugs and to do some sanity checks the right amount of white space is preserved.
But yes, with the updated test fixtures of the original four tests they are practically redundant...